libnavui-dropin/src/main/java/com/mapbox/navigation/dropin/ViewStyleCustomization.kt

```diff
@@ -109,46 +110,40 @@ class ViewStyleCustomization {
     var roadNameTextAppearance: Int? = null
 
     /**
-     * Provide custom [MapboxExtendableButton] style for re-center button.
-     * Use [defaultRecenterButtonStyle] to reset to default.
+     * Provide custom [MapboxExtendableButton] params.
```

Comment:
```suggestion
     * Provide custom parameters for [MapboxExtendableButton] to customize button styles.
```
Same for all the documentation below 👇🏻